docs/topics_5/web-create-project.adoc

```diff
@@ -0,0 +1,111 @@
+// Module included in the following assemblies:
+// * docs/web-console-guide_5/master.adoc
+[id='add_project_{context}']
+= Creating a project
+
+You create a project using the *Create project* wizard.
+
+.Procedure
+
+. In the {WebName}, click *Projects*.
+. Click *Create project*.
++
+.Project details screen
+image::web-add-project_51.png[Project details screen]
+
+. Enter a unique name for the project, an optional description, and click *Next*.
++
+.Applications screen
+image::web-add-apps_51.png[Applications screen]
++
+. To upload applications, click *Browse*, select the application files you want to upload, and click *Close*.
++
+Uploading applications stores them directly in the {ProductShortName} server.
+
+. To register a server path, click the *Server path* tab and enter the *Server-side path* of the application in the field.
++
+Registering the server path of the file tells the {ProductShortName} server the location of the application. This method is useful if you expect to update the application because it ensures that the {ProductShortName} server always uses the latest version of the application.
+
+ . Click *Next*.
++
+.Transformation target screen
+image::web-configure-analysis_51.png[Transformation target screen]
+
+. Click the desired transformation targets.
+. Click *Next*.
++
+You can specify which Java application packages to include or exclude on the *Select applications* screen.
++
+The list on the left contains all of the identified packages in the project. The list on the right contains only the packages that are to be analyzed.
+
+. Select the packages you want to include or exclude and click *>* or *<*.
+. Click *Next*.
++
+.Custom rules screen
+image::web-custom-rules-project_51.png[Custom rules screen]
++
+You can add custom rules to use in each project by uploading the custom rules or registering a server path that contains the rules.
+
+. If you want to add a rule, click *Add rule*.
+. To upload rules, click *Browse*, select a rule set file (`\*.mta.xml` or `*.windup.xml`), and click *Close*.
```

Comment:
Change tl

To upload a ruleset file, click Browse, select a file [can they choose more than one? If so, select one or more files], and click Close.
+
A ruleset file must have an `.mta.xml` or `.windup.xml` extension.

Reply:
The screen opens with the Upload tab open, so I'm omitting that clause.

Reply:
This is one case where I think you can go against the "defaults' rule. Remember that the devs could conceivably add more tabs in the future. Since there are only two tabs and the second tab is mentioned, it's a good idea to mention this tab, even if it is the default.

Reply:
Otherwise, it looks a little weird to someone who is reading the doc without the interface. Since you mention 2nd tab, I would also mention this tab even though it appears by default

Reply:
Also, the devs could change the order of these tabs.